File: Backend/tecnpoint/src/main/java/com/techsolutions/tecnpoint/DTO/UsuarioDTO.java
package com.techsolutions.tecnpoint.DTO;

import com.techsolutions.tecnpoint.enums.TipoUsuario;
import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
public class UsuarioDTO {

    Long idUsuario;
    String nome;
    TipoUsuario tipoUsuario;
}

File: Desktop/TecnPoint/TecnPoint.Modelos/DTO/UsuarioDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.InteropServices;
using System.Text;
using System.Threading.Tasks;
using TecnPoint.Modelos.Enum;

namespace TecnPoint.Modelos.DTO
{
    public class UsuarioDTO
    {
        public long id_usuario { get; set; }
        public string nome { get; set; }
        public TipoUsuario tipoUsuario { get; set; }
    }
}
